### ISSUE TITLE
[feature] Option to not send notification to Action Center

### ISSUE THREAD

**Issue by hackles** [Windos/BurntToast#36]
I'd like to be able to send a notification, have it display for x amount of seconds, and then disappear **without being sent to the Action Center**.

I'm pretty new to PowerShell, so this might be possible with BurntToast and I just wasn't able to figure it out.

I can do this manually using `System.Windows.Forms.NotifyIcon` ([this](https://github.com/proxb/PowerShell_Scripts/blob/master/Invoke-BalloonTip.ps1) + `$balloon.dispose()`), but I'd rather stick with BurntToast because it's a cleaner API.

If this isn't currently possible, can a parameter be added to support it? If it is possible, can someone offer me some guidance on how to do it?

**Comment by Windos** [comment]
Hey, sorry for the late reply. I was just discussing this elsewhere, and that jugged my memory about this issue.

> I'll need to refresh myself on what the relevant registry keys would be (so that you can do it en mass, and so that I can make it part of the module).
> 
> But if you fire up the Settings app,  choose System, then "Notifications & actions".
> 
> Scroll down and you'll see a list of apps that can send notifications, you can drill into any of them (say "Windows PowerShell" if you're using the BurntToast defaults) and turn off "Show notification banners" but leave on "show notifications in action center"
> 
> Unfortunately this means *all* toasts from PowerShell will skip being displayed, rather than it being customizable on a toast by toast basis
> 
> (Though, I may be able to do that via the registry depending on how fast the change takes effect)

I'm going to add this to the roadmap, if changing that setting via the registry is picked up (nearly) instantly, I should be able to expose this as an option when submitting a toast...

**Comment by Windos** [comment]
Hey @hackles, this (or at least something that will have the same effect) has just been added and will be included in v0.7.0

Going forward, you'll be able to set an Expiration Time on a toast. This is a datetime value, after which the toast is removed (from the screen and the action center).

As an example, I tried the following:

```powershell
New-BurntToastNotification -Text 'Expiring Toast Test' -ExpirationTime (Get-Date).AddSeconds(5)
```

This showed the toast, and then removed it from the action center soon after is slid off the screen. Do not that it will be "in" the action center until it's been expired.

An additional feature accomplishes the opposite, a new SuppressPopup switch will mean the toast isn't displayed on screen but *does* go into the action center.


Thanks for opening up this feature request! Keep an eye out for the next release soon(tm)
